pkgs/etc-profiles/default.nix

```diff
@@ -0,0 +1,4 @@
+{
+  capacitated,
+}:
+capacitated.etc-profiles.packages.etc-profiles
```

Comment:
Is this completely unrelated to the propagated built inputs change?

Reply:
Not completely unrelated, in that it's the etc-profiles package that specifically requires the buildenv modifications. That said, this is the equivalent of what `flox register` would do for the `etc-profiles` package, and much the same way that `flox publish` does not require a PR I imagine we could commit this separately (without a PR).